tests/test_routes.py

```diff
@@ -256,14 +256,17 @@ def test_get_inventory_by_query_available(self):
         inventories: List[Inventory] = self._create_inventories(5)
         for inv in inventories:
             count[inv.available] += 1
-        resp = self.app.get(f"{BASE_URL}?available=True",
-                            content_type="application/json")
-        data = resp.get_json()
-        self.assertEqual(len(data), count[True])
-        resp = self.app.get(f"{BASE_URL}?available=False",
-                            content_type="application/json")
-        data = resp.get_json()
-        self.assertEqual(len(data), count[False])
+        for available in {True, False}:
```

Comment:
I think there's no need to use `Set` here. Maybe we can use `List`.

```py
for available in [True, False]:
```

Reply:
Good catch. Thanks!